### ISSUE TITLE
Improve XML file handling, update to latest FEBio and febio_spec 2.5

### ISSUE THREAD

**Issue by Kevin-Mattheus-Moerman** [gibbonCode/GIBBON#19]
The latest FEBio version offers many new features. Some require the febio_spec 2.5 format. Currently GIBBON is coded based on febio_spec 2.0. It would be good to update to 2.5. 
In parallel the feb file (XML file) handling is difficult to expand and involves many dedicated functions. It would be better to have a single clean function to handle XML file generation which can cope with arbitrary XML components (and is therefore compatible with any febio_spec).

**Comment by Kevin-Mattheus-Moerman** [comment]
This functionality has been added with c1cdbb2. The documentation file `HELP_febioStruct2xml` gives an overview of the capabilities. Next new `DEMO_FEBio...` files will need to be prepared to illustrate the use with febio_spec 2.5. I'll open a new issue for this.
